### PR TITLE
Update baseline statuses

### PR DESCRIPTION
**Pull request by ddbeck**
I've gotten a bit better at (manually) exercising the baseline status, so I went through everything we already had to make sure it was correct. Some changes fell out of that. I'll self-review with detailed commentary.

### CODE REVIEW

File: feature-group-definitions/canvas-context-lost.yml [hunk 1]
```diff
@@ -5,7 +5,6 @@ status:
   support:
     chrome: "99"
     edge: "99"
-    firefox: "105"
```
Comment (ddbeck) [inline feature-group-definitions/canvas-context-lost.yml:8]: Firefox supports only the events on `OffscreenCanvas` and none of the rest of the API.

File: feature-group-definitions/container-queries.yml [hunk 2]
```diff
@@ -4,5 +4,17 @@ usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/4165
...
+  - api.CSSContainerRule
+  - api.CSSContainerRule.containerName
+  - api.CSSContainerRule.containerQuery
+  - css.at-rules.container
+  - css.properties.contain
+  - css.properties.contain.inline-size
+  - css.properties.contain.style
+  - css.properties.container
+  - css.properties.container-name
+  - css.properties.container-type
```
Comment (ddbeck) [inline feature-group-definitions/container-queries.yml:20]: I don't know if this feature list is good, to be honest. Safari is missing the CSSOM bit. If we relaxed this requirement, then we'd be looking at Safari 17 as the baseline-effective version.

File: feature-group-definitions/cascade-layers.yml [hunk 3]
```diff
@@ -3,9 +3,10 @@ caniuse: css-cascade-layers
 usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/4007
 status:
   is_baseline: true
-  since: 2022-03-15
+  since: "2022-09-19"
```
[the remark this answers is not in the document]
Comment (foolip) [reply feature-group-definitions/cascade-layers.yml:6]: Ah, that's a curious property of baseline dates I hadn't thought about. It's right, but I think takes a bit of explanation if these dates ever are shown to developers.

File: feature-group-definitions/container-queries.yml (hunk 2 above)
Comment (foolip) [reply feature-group-definitions/container-queries.yml:20]: I think it's fine to not consider CSSOM part of CQ, it matters very little in practice.

Comment (ddbeck) [reply feature-group-definitions/container-queries.yml:20]: OK, in that case this is a coming-soon feature, which we don't want to show just yet. I'll drop the CSSOM bits from the feature list but update-and-comment-out the baseline data.

**Pull request merged**